### PR TITLE
Change schema to start to match up with the client.

### PR DESCRIPTION
**Pull request by jhugman**
@mcomella's https://github.com/mozilla-mobile/prox-server/pull/8 is now unmergable.

There are a bunch of questions in the review comments. Uncontraversial changes that come out of those comments are in this Pull Request.

### CODE REVIEW

File: app/representation.py [hunk 1]
```diff
@@ -3,37 +3,44 @@ def venueRecord(biz, **details):
...
-        "reviewCount": biz.review_count,
+        "totalReviewCount": biz.review_count,
         "ratingMax": 5,
-        "description": biz.snippet_text,
+        "summary": biz.snippet_text,
         "url": biz.url
       }
       images = images + info["photos"]
       hours = info["hours"][0]["open"]
+      categories |= set([ c["title"] for c in info["categories"] if "title" in c])
```
Comment (mcomella) [inline app/representation.py]: Does Yelp provide categories sorted by any priority? If so, I think it'd be great to preserve that priority on the client.

**Comment by mcomella** [comment]
[Wire up Places with Place Carousel Cards](https://trello.com/c/fDEYEBI3/119-wire-up-places-with-place-carousel-cards)

**Comment by jhugman** [comment]
[As a user I can see images from multiple sources so I can get an idea what a venue looks like.](https://trello.com/c/pihBo72r/137-as-a-user-i-can-see-images-from-multiple-sources-so-i-can-get-an-idea-what-a-venue-looks-like)

File: app/representation.py [hunk 2]
```diff
@@ -1,43 +1,132 @@
...
       info = details["yelp"]
       providers["yelp"] = {
-        "rating": biz.rating,
-        "reviewCount": biz.review_count,
-        "ratingMax": 5,
-        "description": biz.snippet_text,
-        "url": biz.url
+        "rating"          : biz.rating,
+        "totalReviewCount": biz.review_count,
+        "ratingMax"       : 5,
```
Comment (liuche) [inline app/representation.py:24]: The schema doesn't have a ratingMax, and instead has rating as a "normalized 0-5 w/ half values"

**Review by liuche — APPROVED** [review]
Just one comment, but we can address it in a followup, either as a change to the schema or removing the extraneous field.

**Pull request merged**